gis/gdal/Portfile

```diff
@@ -10,7 +10,7 @@ PortGroup           compiler_blacklist_versions 1.0
 mpi.setup
 
 github.setup        OSGeo gdal 2.3.2 v
-revision            5
+revision            6
```

Comment:
Please keep the `revision` back at 5. Adding a variant does not change any installed files for those with this port installed. One must re-install the port to get the new variant.